packages/react-components/src/components/styles/RichTextEditor.styles.ts

```diff
@@ -308,3 +308,10 @@ export const insertTableMenuTitleStyles = mergeStyles({
   fontSize: '0.75rem',
   marginBottom: '0.5rem'
 });
+
+/**
+ * @private
+ */
+export const tableContextMenuIconStyles = mergeStyles({
+  marginTop: '6px'
```

Comment:
I know we don't have clear answer yet for px/rem but for consistency, should we put this in rem?

Reply:
Left it px because of this :) sure, will update!